node_modules/oae-ui/tests/test-ui.js

```diff
@@ -561,6 +561,56 @@ describe('UI', function() {
                 });
             });
         });
+
+         /**
+         * Test that verifies that a new logo can be uploaded for a tenant and a signed URL is returned
+         */
+        it('verify logo can be uploaded for a tenant', function(callback) {
+            var tenantAlias = global.oaeTests.tenants.cam.alias;
+            var fileStream = fs.createReadStream(__dirname + '/data/oae-logo.png');
```

Comment:
A quick suggestion: Since `__dirname + '/data/oae-logo.png'` is repeated several times in this test, why not create a local variable for the file path and re-use it?